### PR TITLE
Use common library function for getting the module name

### PR DESCRIPTION
**Pull request by jsturtevant**
Quick follow up to #219 and #156

### CODE REVIEW

**Review by jprendes — APPROVED** [review]
LGTM!

File: crates/containerd-shim-wasmedge/src/executor.rs [hunk 1]
```diff
@@ -55,15 +55,12 @@ impl Executor for WasmEdgeExecutor {
...
-        }
-
-        let start = args[0].clone();
-        let mut iterator = start.split('#');
-        let cmd = iterator.next().unwrap().to_string();
-        let path = PathBuf::from(cmd);
+        let (module_name, _method) = oci::get_module(spec);
+        let module_name = match module_name {
+            Some(m) => m,
+            None => return false,
```
Comment (Mossaka) [inline crates/containerd-shim-wasmedge/src/executor.rs:61]: Could you please add a `debug!()` or `info!()` here for logging why can_handle returns a false?

**Review by Mossaka — APPROVED** [review]
LGTM! Left one minor comment.

**Review by Mossaka — APPROVED** [review]

**Comment by Mossaka** [comment]
Ill merge once the CI passes.

**Pull request merged**